website/client/js/services/questServices.js

```diff
@@ -68,9 +68,10 @@ angular.module('habitrpg')
         }
       }
       text += '---\n\n';
-      text += '**' + window.env.t('rewards') + ':**\n\n';
-      if(quest.drop.items) {
-        for (var item in quest.drop.items) {
+      text += '**' + window.env.t('rewardsAllParticipants') + ':**\n\n';
+      var participantRewards = quest.drop.items ? _.reject(quest.drop.items, 'onlyOwner') : quest.drop.items;
+      if(participantRewards) {
+        for (var item in participantRewards) {
```

Comment:
Let's use `_.each` here. `for in` loops have some bad behavior when the Object.prototype is modified by a third party. (I know you didn't write the original `for in`, but we should still clean it up.